Calculate/Calculate/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Calculate;
+
+public class Program
+{
+    public Action<string?> WriteLine
+    {
+        get;
+        init;
+    } = s => Console.WriteLine(s);
+
+    public Func<string?> ReadLine
+    {
+        get;
+        init;
+    } = () => Console.ReadLine();
+
+    static void Main()
+    {
+        Program program = new();
+        Calculator calculator = new();
+        bool done = false;
+
+        program.WriteLine("Please Enter a Single-Operation Calculation" +
+            "Typing exit will end the program. \n");
+
+        do
+        {
+            string? input = program.ReadLine();
+            if(input != null && input.Equals("exit", StringComparison.Ordinal))
+            {
+                done = true;
+            }
+            else if(input != null && Calculator.TryCalculate(input, out double output))
```

Comment:
```suggestion
            else if(input != null && calculator.TryCalculate(input, out double output))
```
By using `Calculator` here, your instance `calculator` technically goes unused.